src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx

```diff
@@ -115,11 +115,15 @@ const AspectDashboardContainer: FC<AspectDashboardContainerProps> = ({
       createdAt: new Date(x.timestamp),
     }));
 
+  const isAuthor = useCallback(
```

Comment:
There's no point in wrapping functions in `useCallbacks` unless you pass them as props directly.